deluge/plugins/WebUi/deluge_webui/core.py

```diff
@@ -75,6 +86,7 @@ def start_server(self):
 
         self.server.port = self.config['port']
         self.server.https = self.config['ssl']
+        self.server.setup_webapidoc(enabled=self.config['webapidoc_enabled'])
```

Comment:
I don't this setup should be part of the plugin code, the server itself should check config and do the setup